WordPress/WordPressTest/MediaServiceRemoteRESTTests.swift

```diff
@@ -0,0 +1,144 @@
+import XCTest
+@testable import WordPress
+
+class MediaServiceRemoteRESTTests: XCTestCase {
+
+    let mockRemoteApi = MockWordPressComRestApi()
+    var mediaServiceRemote: MediaServiceRemoteREST!
+    let siteID = NSNumber(integer: 99999)
```

Comment:
I think you can just put `let siteID = 99999` and Swift will automatically do the casting.